pkg/cli/admin/release/extract.go

```diff
@@ -98,8 +98,13 @@ func NewExtract(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 
 			# Extract cloud credential requests for AWS
 			oc adm release extract --credentials-requests --cloud=aws
+
+			# Extract a specifc version and cpu architecture
+			# Default architecture is the amd64
+			oc adm release extract 4.10.8 --to=/tmp/example --release-arch=arm64
```

Comment:
reason i preferred `--graph-arch` is that - `release-arch` seems like it denotes the architecture of a release. whereas what this is trying to do is get the arch and release version and fetch the corresponding graph.

Reply:
I agree, that `release-arch` isn't great, but not sure if a user would understand `graph-arch` either? @andymcc @yselkowitz  do you guys have any thoughts here?